shanoir-ng-front/src/app/shared/components/table/table.component.ts

```diff
@@ -185,6 +188,11 @@ export class TableComponent implements OnInit, OnChanges, OnDestroy {
         else if (this.selectionAllowed) this.onSelectChange(item, !this.isSelected(item));
     }
 
+    downloadStats(item: any) {
```

Comment:
I find it quite strange to put this specific method in the really generic [table.component.ts](https://github.com/fli-iam/shanoir-ng/pull/2416/files#diff-2b99fc510aa1b2f470bbec831bc9d3e4ccb0e512e9045ef42e9748284bf2076d) component.
Is it possible to put it in async-tasks.component.ts for example ?